style.css

```diff
@@ -0,0 +1,285 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+html {
+  overflow-x: hidden;
+}
+
+body {
+  background-color: #f4f5f6;
+}
+
+a {
+  text-decoration: none;
+}
+
+/* desktop navbar */
+
+.top_nav {
+  background-color: #f72210;
+  height: 95px;
+  align-items: flex-end;
+}
+
+.top_nav > div > img {
+  width: 366px;
+  height: 85px;
+  cursor: pointer;
+}
+
+.top_nav > div {
+  margin-bottom: 10px;
+}
+
+.top_nav > p {
+  color: white;
+  font-size: 15px;
+}
+
+.nav_bar_mid {
+  width: 100%;
+  height: 54px;
+}
+
+.nav_bot1 > a {
+  font-size: 14px;
+  color: #666;
+}
+
+.nav_bar_mid > li {
+  display: flex;
+  justify-content: center;
+  margin: auto;
+  width: 100%;
+  border-left: 1px solid #d9d9d9;
+}
+
+.nav_bar_mid > li > a {
+  color: black;
+  font-weight: bold;
+}
+
+.search_part {
+  gap: 10px;
+  align-items: center;
+}
+
+#border_red {
+  border-top: 20px solid red;
+  border-left: 20px solid white;
+}
+
+.last_nav {
+  height: 46px;
+  width: 100%;
+  display: flex;
+  border-top: 1px solid #f4f5f7;
+  border-bottom: 1px solid #f4f5f7;
+}
+
+.nav_bar_bot {
+  height: 100%;
+  width: 100%;
+}
+
+.nav_bar_bot > li {
+  margin: auto;
+}
+
+.nav_bar_bot > li > a {
+  color: #807f84;
+}
+
+.text_deco > li > a {
+  color: black;
+  text-decoration: none;
+}
+
+.nav_bot2 > ul > li > a {
+  font-size: 14px;
+  color: #666;
+}
+
+.border_top > div > a:hover {
+  color: black;
+  text-decoration: none;
+}
+
+.nav_bar_mid > li > a:hover {
+  color: red;
+  text-decoration: none;
+}
+
+.nav_bar_bot > li > a:hover {
+  color: red;
+  text-decoration: none;
+}
+
+/* phone navbar */
+
+#phone_navbar {
+  background-color: #f72210;
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  height: 50px;
+  position: sticky;
+  top: 0;
+}
+
+#phone_cont {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
```

Comment:
- [ ]  Use Bootstrap for Display Flex.
- [ ]  Use Bootstrap for aligning Items.
- [ ] Use Bootstrap for justifying content.